cineast-api/src/main/java/org/vitrivr/cineast/api/APIEndpoint.java

```diff
@@ -88,7 +125,14 @@ private static Service dispatchService(boolean secure) {
             service.webSocket(String.format("/%s/%s/websocket", CONTEXT, VERSION), WebsocketAPI.class);
         }
 
+        /* Init Swagger */
+        if(config.getEnableLiveDoc() && swagger == null){
+            swagger = SparkSwagger.of(service).ignores(IgnoreSpec.newBuilder().withIgnoreAnnotated(
+                JsonIgnore.class)::build).endpoint(instance);
+        }
 
+/*
+        // TODO re-implement this functionality
```

Comment:
This should be rather straight forward using the content serving functionality which is also used for multimedia objects and thumbnails.

Reply:
Thanks